src/components/ProfileForm/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import { reduxForm, Field } from 'redux-form/immutable';
+import PropTypes from 'prop-types';
+
+import CustomInput from '../common/CustomInput';
+import CustomButton from '../common/CustomButton';
+import { validations, editProfile } from '../../utils/constraints';
+import { black } from '../../constants/styleConstants';
+import styles from './styles';
+
+const ProfileForm = ({
+  handleSubmit, error, submitting, editProfile
+}) => (
+  <View>
+    <Field
+      name="username"
+      inputLabel="Username"
+      inputType="username"
+      component={CustomInput}
+    />
+    <Field
+      name="email"
+      inputLabel="Email"
+      inputType="email"
+      component={CustomInput}
+    />
+    <Field
+      name="password"
+      inputLabel="Password"
+      inputType="password"
+      component={CustomInput}
+    />
+    <CustomButton
+      title="SAVE CHANGES" color={black}
```

Comment:
`color` prop should be in a new line.

When all the props don't fit in one line you should use one line per prop